src/Microsoft.Azure.SignalR.Common/ClientInvocation/ClientResultsManager.cs

```diff
@@ -0,0 +1,312 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NET7_0_OR_GREATER
```

Comment:
Do we really need to limit this to NET7_0? It seems below NET7_0 doesn't have chance to invoke this. Please check if possible to remove this to make code clean.